pkg/newlog/cmd/newlogd.go

```diff
@@ -656,6 +656,11 @@ func getMemlogMsg(logChan chan inputEntry, panicFileChan chan []byte) {
 			pidStr = strconv.Itoa(logInfo.Pid)
 		}
 
+		// not to upload 'kube' container logs, one can find in /persist/kubelog for detail
+		if logInfo.Source == "kube" {
+			continue
+		}
```

Comment:
Is this required since they are much more voluminous than other logs?

If we need filtering (on more that loglevel) in general should we add some configurable filter to EVE-OS based on Source and other fields?

Reply:
Yeh, last time i looked at my device logs, since last commit of kubevirt missed this in PR, it probably is more than 60% of the total logs from the device. So for now I'll just block this. When the new changes coming for the 'newlog', maybe a controller side config can add new knobs. We can later introduce some configitem also if needed to control this 'kube' container logs.